mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/MapInstance.java

```diff
@@ -417,18 +429,35 @@ public View onCreateView() {
         handler.postDelayed(new Runnable() {
             @Override
             public void run() {
-                oThis.getCamera();
                 // MAP is ready core and engine cameras are synced, so calculate the initial bounds.
                 BoundsGeneration.initialize(MapInstance.this);
+                ICamera tempCamera = MapInstance.this.getCamera();
+                tempCamera.setAltitude(100000.0);
+                MapInstance.this.setCamera(tempCamera, false);
```

Comment:
If we want to start with this altitude then it might better to set the camera in first timeout, start another delayed action and there initialize bounds and fire the MAP_READY.
First timeout could be 100 ms and second could be 300 ms.